### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by vladimyr**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Format badges section, use https://badgen.net as badge provider. :rocket:

**Does this PR introduce a breaking change?**

No.

**Please check if the PR fulfills these requirements**
- [x] The commit message follows semantic-release [guidelines](https://github.com/semantic-release/semantic-release#commit-message-format)
- [ ] Fix/Feature: Docs have been added/updated
- [ ] Fix/Feature: Tests have been added; existing tests pass

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
+    <img src="https://badgen.net/npm/v/vue-analytics">
+  </a>
+  <a href="https://npm.im/vue-analytics">
+    <img src="https://badgen.net/npm/dm/vue-analytics">
+  </a>
+  <a href="https://travis-ci.org/MatteoGabriele/vue-analytics">
+    <img src="https://badgen.net/travis/MatteoGabriele/vue-analytics/master">
+  </a>
+  <a href="https://bundlephobia.com/result?p=vue-analytics">
+    <img src="https://badgen.net/bundlephobia/minzip/vue-analytics">
```
Comment (MatteoGabriele) [inline README.md]: as we did in gtag one same mingzip thing?
thanks again for you work

Comment (vladimyr) [reply README.md]: I did that already, url resolves to minzipped size badge.

**Comment by MatteoGabriele** [comment]
thanks a lot

**Pull request merged**